src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -105,6 +105,7 @@
     private NotificationLevel defaultNotificationLevel = NotificationLevel.UNKNOWN;
     private MFALevel mfaLevel = MFALevel.UNKNOWN;
     private ExplicitContentLevel explicitContentLevel = ExplicitContentLevel.UNKNOWN;
+    private NSFWLevel nsfwLevel = NSFWLevel.UNKNOWN;
```

Comment:
I would assume `NSFWLevel.DEFAULT` makes more sense here since... well... it's the default a Server has...

But this is something Minn should give his input for I would say...

Reply:
Since the `MFALevel` here is also UNKNOWN, i think this is pretty much expected.

Reply:
UNKNOWN is better. We don't want to incorrectly classify a guild as safe or otherwise.